core/includes/filetransfer/filetransfer.inc

```diff
@@ -16,6 +16,16 @@ abstract class FileTransfer {
   protected $hostname = 'localhost';
   protected $port;
 
+  /**
+   * @var string
+   */
+  protected $jail;
+
+  /**
+   * @var string
+   */
+  protected $chroot;
+
```

Comment:
Same question as the previous comment here ...and elsewhere in this PR.

Reply:
Same answer. :wink: It may seem pedantic, to not even fix whitespace or a comment, that's not part of the required code change in this PR, but... 82 files changed, more than 1k lines added – I'm trying to be nice to code reviewers, so you can concentrate on the actual changes without distraction.